test/html/normalize.vows.js

```diff
@@ -85,5 +85,11 @@ wows.describe('html normalizer').addBatch({
     	
     	var dom = new DOMParser().parseFromString('<textarea>alert(a<b&&c?"<br>":">>");</textarea>','text/xml');
     	assert(dom+'', '<textarea>alert(a&lt;b&amp;&amp;c?"<br/>":">>");</textarea>');
+	},
+    'Euro characters': function () {
+        var dom = new DOMParser().parseFromString('<div>&Auml;&auml;&Aring;&aring;&AElig;&Ouml;&ouml;&Oslash;&euro;</div>','text/html');
+        // For the future, it may be nicer to use \uxxxx in the assert strings
+        // rather than pasting in multi-byte UTF-8 Unicode characters
+        assert(dom+'', '<div xmlns="http://www.w3.org/1999/xhtml">ÄäÅåÆÖöØ€</div>');
```

Comment:
Just completed the list.
```suggestion
        var dom = new DOMParser().parseFromString('<div>&Auml;&auml;&Aring;&aring;&AElig;&aelig;&Ouml;&ouml;&Oslash;&oslash;&Uuml;&uuml;&euro;</div>','text/html');
        // For the future, it may be nicer to use \uxxxx in the assert strings
        // rather than pasting in multi-byte UTF-8 Unicode characters
        assert(dom+'', '<div xmlns="http://www.w3.org/1999/xhtml">ÄäÅåÆæÖöØøÜü€</div>');
```